config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
   root to: "root#index"
   resources :users, only: [:new, :create]
+  resources :sessions, only: [:new, :create, :destroy]
   get "/signup", to: "users#new", as: :signup
+  get '/login', to: 'sessions#new', as: :login
+  delete '/logout', to: 'sessions#destroy', as: :logout
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.